src/plugins/data/public/search/search_source/create_search_source.test.ts

```diff
@@ -43,94 +43,50 @@ describe('createSearchSource', () => {
       get: jest.fn().mockReturnValue(Promise.resolve(indexPatternMock)),
     } as unknown) as jest.Mocked<IndexPatternsContract>;
 
-    createSearchSource = createSearchSourceFromJSON(indexPatternContractMock, dependencies);
+    createSearchSource = createSearchSourceFactory(indexPatternContractMock, dependencies);
   });
 
-  test('should fail if JSON is invalid', () => {
```

Comment:
Not a big deal but seems like this test got lost - looks like it should become a unit test of https://github.com/elastic/kibana/pull/64207/files#diff-0138b68230bd9fd9e68b55fe4e8868b1